STDentalSolution/src/STDentalLibrary/Models/ModelsResponse/UserInfo.cs

```diff
@@ -10,8 +10,8 @@ public class UserInfo
     {
         public int Id { get; set; }
 
-        public string User { get; set; }
+        public string? User { get; set; }
```

Comment:
nullable reference types is a variant of C# so may be not obvious, I would not recommend to use it